src/wallet/rpc/transactions.cpp

```diff
@@ -436,7 +447,11 @@ static const std::vector<RPCResult> TransactionDescriptionString()
            {RPCResult::Type::NUM_TIME, "timereceived", "The time received expressed in " + UNIX_EPOCH_TIME + "."},
            {RPCResult::Type::STR, "comment", /*optional=*/true, "If a comment is associated with the transaction, only present if not empty."},
            {RPCResult::Type::STR, "bip125-replaceable", "(\"yes|no|unknown\") Whether this transaction could be replaced due to BIP125 (replace-by-fee);\n"
-               "may be unknown for unconfirmed transactions not in the mempool."}};
+               "may be unknown for unconfirmed transactions not in the mempool."},
+           {RPCResult::Type::ARR, "wallet_descs", /*optional=*/true, "Only if 'category' is 'received'. List of imported descriptors matching the scriptPubKey of this coin.", {
```

Comment:
In 8976cd89051d262bfed2498de5aa9ed190e3b331 "rpc: output wallet descriptors for received entries in listsinceblock"

Not just import descriptors